Tensile/Contractions.py

```diff
@@ -63,6 +63,11 @@ def FromOriginalState(cls, d):
         batchIndices = []
         boundIndices = []
 
+        try:
+            cls.convolution = d.convolution
```

Comment:
`cls` here is the *class*, not any instance.  This is like

ProblemType.convolution = d.convolution.

The instance is `rv`.